docs/index.md

```diff
@@ -26,52 +34,48 @@ for more details.
 - Usage instructions:
   - [Zephyr](readme-zephyr.md)
-  - [Mynewt](readme-mynewt.md)
+  - [Apache Mynewt](readme-mynewt.md)
+  - [Apache NuttX](readme-nuttx.md)
   - [RIOT](readme-riot.md)
-  - [Mbed-OS](readme-mbed.md)
-- [Patch submission](SubmittingPatches.md) - information
-  on how to contribute to MCUboot
+  - [Mbed OS](readme-mbed.md)
+  - [Espressif IDF](readme-espressif.md)
 - Testing
   - [Zephyr](testplan-zephyr.md) test plan
-  - [mynewt](testplan-mynewt.md) test plan
+  - [Mynewt](testplan-mynewt.md) test plan
+- [Patch submission](SubmittingPatches.md) - information
+  on how to contribute to MCUboot
 - [Release process](release.md)
 
-There is also a document about [signed images](signed_images.md) that is out
-of date.  You should use `imgtool.py` instead of these documents.
+There is also a document about [signed images](signed_images.md) that is
+currently outdated. Follow the instructions in [imgtool](imgtool.md) instead.
 
 ## Roadmap
 
-The issues being planned and worked on are tracked using GitHub issues. To participate
-please visit:
-
-[MCUboot Issues](https://github.com/mcu-tools/mcuboot/issues)
-
-~~Issues were previously tracked on [MCUboot JIRA](https://runtimeco.atlassian.net/projects/MCUB/summary)
-, but it is now deprecated.~~
+The issues being planned and worked on are tracked using GitHub issues. To
+give your input, visit [MCUboot GitHub
+Issues](https://github.com/mcu-tools/mcuboot/issues).
 
 ## Browsing
 
-Information and documentation on the bootloader is stored within the source.
-
-~~It was previously also documented on confluence: [Confluence page](https://runtimeco.atlassian.net/wiki/discover/all-updates)
-, but it is now deprecated and not currently maintained~~
-
-For more information in the source, here are some pointers:
-
+You can find additional documentation on the bootloader in the source files.
+For more information, use the following links:
 - [boot/bootutil](https://github.com/mcu-tools/mcuboot/tree/main/boot/bootutil): The core of the bootloader itself.
 - [boot/boot\_serial](https://github.com/mcu-tools/mcuboot/tree/main/boot/boot_serial): Support for serial upgrade within the bootloader itself.
-- [boot/zephyr](https://github.com/mcu-tools/mcuboot/tree/main/boot/zephyr): Port of the bootloader to Zephyr
-- [boot/mynewt](https://github.com/mcu-tools/mcuboot/tree/main/boot/mynewt): Mynewt bootloader app
-- [boot/mbed](https://github.com/mcu-tools/mcuboot/tree/main/boot/mbed): Port of the bootloader to Mbed-OS
+- [boot/zephyr](https://github.com/mcu-tools/mcuboot/tree/main/boot/zephyr): Port of the bootloader to Zephyr.
+- [boot/mynewt](https://github.com/mcu-tools/mcuboot/tree/main/boot/mynewt): Bootloader application for Apache Mynewt.
+- [boot/nuttx](https://github.com/mcu-tools/mcuboot/tree/main/boot/nuttx): Bootloader application and port of MCUboot interfaces for Apache NuttX.
+- [boot/mbed](https://github.com/mcu-tools/mcuboot/tree/main/boot/mbed): Port of the bootloader to Mbed OS.
```

Comment:
Let's add Cypress here as well.

Reply:
The Espressif port is also missing here:
```suggestion
- [boot/mbed](https://github.com/mcu-tools/mcuboot/tree/main/boot/mbed): Port of the bootloader to Mbed OS.
- [boot/espressif](https://github.com/mcu-tools/mcuboot/tree/main/boot/espressif): Bootloader application and MCUboot port for Espressif SoCs.
```

Reply:
fixed